### ISSUE TITLE
[A11y] General - Login modal is inaccessible

### ISSUE THREAD

**Issue by jechols** [OregonDigital/OD2#1790]
### Operating System

Windows

### Browser

Edge

### Affected page

Advanced Search

### Admin user?

- [ ] Yes

### Issue Description

(Note that this is probably a site-wide problem, not just advanced search - it just happens that's where I am testing right now)

While trying to log in as admin I found that the login modal is in fact inaccessible.  Tabbing causes focus to disappear for some reason before it gets back to the beginning of the modal.  That's potentially a fail, but it's minor.  A much bigger problem is that none of the radio buttons have labels.

### Suggested Fix

- Make sure each tab goes somewhere that's visible for keyboard-only users.  I don't know if the "hidden" focusable element is meaningful or an accident of JS or what.
- Give all radio buttons valid labels.  This is critical for screen reader users as well as improving the "hit area" for tablet or phone users.
